unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/IAvatarSlotComponentView.cs

```diff
@@ -19,5 +21,6 @@ public interface IAvatarSlotComponentView
         void SetHideList(string[] hideList);
         string[] GetHideList();
         void ShakeAnimation();
+        void SetHideVisible(bool isVisible);
```

Comment:
This name is a bit confussing... can we change it by something more clear like `SetVisible`?